runtime/nls/dump/j9dmp.nls

```diff
@@ -157,6 +157,16 @@ J9NLS_DMP_WRITTEN_DUMP_STR.link=
 
 # END NON-TRANSLATABLE
 
+J9NLS_DMP_WRITTEN_DUMP_STDOUTERR_STR=%1$s dump written
+# START NON-TRANSLATABLE
+J9NLS_DMP_WRITTEN_DUMP_STR.explanation=The dump was written to STDOUT/ERR.
+J9NLS_DMP_WRITTEN_DUMP_STR.system_action=The JVM continues.
+J9NLS_DMP_WRITTEN_DUMP_STR.user_response=To understand how to interpret a java dump file, refer to diagnostics information.
+J9NLS_DMP_WRITTEN_DUMP_STR.sample_input_1=Java
+J9NLS_DMP_WRITTEN_DUMP_STR.link=
```

Comment:
Given @pshipton's suggestion to just use `/STDOUT/` or `/STDERR/` for the 'file name', I don't think we need or want a new NLS message.